LICENSE.md

```diff
@@ -35,8 +39,50 @@ and
 
     WordPress is released under the GPL
 
+
+### Dual License 
+
+**We are currently in the process of changing Gutenberg’s software license from
+GPL (or later) to a dual license: GPL (or later) and MPL.**
```

Comment:
May be fine to remove both instances of `(or later)` here as the "GPL" defining section above already notes v2 or later.

Reply:
Good catch! I made the change.